pdns/dnsname.hh

```diff
@@ -376,3 +378,11 @@ bool DNSName::operator==(const DNSName& rhs) const
 }
 
 extern const DNSName g_rootdnsname, g_wildcarddnsname;
+
+struct DNSNameSet: public std::set<DNSName> {
```

Comment:
Why not an std::unordered_set? This is almost always way faster.

Reply:
Right you are, I was in a hurry. Fixed.